submissions/Vas-P/popup/css/style.css

```diff
@@ -115,39 +133,42 @@ nav{
 	background: #555;
 }
 
-.hidden-items, .more-items{
+.hidden_items, .more_items{
 	display: flex;
 	flex-wrap: wrap;
 	justify-content: space-around;
 }
 
-.hidden-items > li, .more-items > li{
+.hidden_items > li, .more_items > li{
 	margin-top: 10px;
 	width: 90px;
 	text-align: center;
 }
 
-.hidden-items > li > a:hover, .more-items > li > a:hover{
+.hidden_items > li > a:hover,
+.more_items > li > a:hover{
 	border: 1px solid #989898;
+	border-radius: 2px;
 }
 
-.hidden-items > li > a > img, .more-items > li > a > img{
-	width: 70px;
+.hidden_items > li > a > img, .more_items > li > a > img{
```

Comment:
Avoid creating too specific selectors. In a big project, it's hard to support or override by another selector.
Use `class` for adding styles to element.
[Read more about selector specificity.](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity)

Reply:
Added class for img's, thx for review.